src/Interfaces/AbstractHyperrectangle.jl

```diff
@@ -555,7 +555,7 @@ A list of `Hyperrectangle`s.
 function split(H::AbstractHyperrectangle{N}, num_blocks::AbstractVector{Int}
               ) where {N}
     @assert length(num_blocks) == dim(H) "need number of blocks in each dimension"
-    radius = copy(radius_hyperrectangle(H))
+    radius = copy_vector(radius_hyperrectangle(H))
```

Comment:
This is OK but I suggest to use library functions when possible:

```julia
R = radius_hyperrectangle(H)
radius = similar(R)
copyto!(radius, R)
```
Returns an MArray if `R` is a static array.